src/items/withCaption.tsx

```diff
@@ -1,15 +1,41 @@
 import TextDisplay from '@/TextDisplay/TextDisplay';
 
-import { Box, Stack } from '@mui/material';
+import { Stack } from '@mui/material';
 
-import { DescriptionPlacement, DescriptionPlacementType } from '@graasp/sdk';
+import {
+  DescriptionPlacement,
+  DescriptionPlacementType,
+  UnionOfConst,
+} from '@graasp/sdk';
 
 const DEFAULT_ITEM_DESCRIPTION = '';
 
+export const Alignment = {
+  Center: 'center',
+  Left: 'left',
+  Right: 'right',
+} as const;
+export type AlignmentType = UnionOfConst<typeof Alignment>;
+
+const getAlignItemsFromAlignmentSetting = (
+  alignment: AlignmentType,
+): 'flex-start' | 'flex-end' | 'center' => {
+  switch (alignment) {
+    case Alignment.Right:
+      return 'flex-end';
+    case Alignment.Center:
+      return 'center';
+    case Alignment.Left:
+    default:
+      return 'flex-start';
+  }
+};
```

Comment:
This is experimental and most likely is going to be moved to sdk when implementing the feature in the builder.

Reply:
add in comments.